dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/trackedentity/SecurityOwnershipValidator.java

```diff
@@ -91,14 +95,18 @@ public void validate(
                 .getOrganisationUnit()
             : bundle.getPreheat().getOrganisationUnit(trackedEntity.getOrgUnit());
 
-    // If trackedEntity is newly created, or going to be deleted, capture
-    // scope has to be checked
-    if (strategy.isCreate() || strategy.isDelete()) {
+    // If trackedEntity is newly created, capture scope has to be checked
+    if (strategy.isCreate()) {
+      checkTeiTypeWriteAccess(reporter, bundle, trackedEntity, trackedEntityType);
       checkOrgUnitInCaptureScope(reporter, bundle, trackedEntity, organisationUnit);
     }
-    // if its to update trackedEntity, search scope has to be checked
+    // if its to update or delete trackedEntity, write access has to be checked
```

Comment:
similar to the one above, I think we either expand on why write access has to be checked, or we remove the comment altogether.